lib/update.js

```diff
@@ -18,23 +19,22 @@ function cmd(args, cb) {
     .catch(cb)
 }
 
-async function update (args) {
+const update = async args => {
   const update = args.length === 0 ? true : args
   const where = npm.flatOptions.global
     ? globalTop
     : npm.prefix
-  
-  const arb = new Arborist({ 
-    ...npm.flatOptions, 
-    path: where 
+
+  if (npm.flatOptions.depth !== Infinity) {
+    log.warn('update', 'The --depth option no longer has any effect. See RFC0019.\n' +
+      'https://github.com/npm/rfcs/blob/latest/accepted/0019-remove-update-depth-option.md')
```

Comment:
I'd prefer to be even more cautious and link to: https://github.com/npm/rfcs/blob/ea2d3024e6e149cd8c6366ed18373c9a566b1124/accepted/0019-remove-update-depth-option.md instead

Reply:
Hm! I considered that, but I figured if we ever update it or something, we'd want to have the user see the latest and greatest version, no?

Reply:
I would still prefer linking to the commit as I can see many possible problematic things happening:
- what if us (or a future team working on this repo) decides to rename branches? `latest` to `master`, etc
- what if someone at a point in future decides to rename the `accepted` folder?
- what if that rfc gets withdrawn and/or moved to a diff folder?

Although it seems unlikely in the near-future, I had been bitten by these sorts of changes in the past and I grew to really appreciate linking to the commit blob links instead, just by _seeing_ these things actually happening 😊 

but then again I also see that the chance of this message changing before any of the aforementioned happening is also really high 😄